elfi/visualization/visualization.py

```diff
@@ -99,7 +99,8 @@ def _create_axes(axes, shape, **kwargs):
     else:
         fig, axes = plt.subplots(ncols=shape[1], nrows=shape[0], **fig_kwargs)
         axes = np.atleast_2d(axes)
-        fig.tight_layout(pad=2.0)
+        fig.tight_layout(h_pad=0.0, w_pad=0.0)
+        fig.subplots_adjust(wspace=0.1, hspace=0.1)
```

Comment:
i think the titles were better positioned with `pad=2.0` when `plot_marginals` is used to visualise smc results

Reply:
True! Will change paddings accordingly.